code/game/objects/items/devices/ioncannon_remote.dm

```diff
@@ -0,0 +1,54 @@
+
+
+/obj/item/device/loic_remote
+	name = "\improper strange remote"
+	desc = "Press the big button to upset people."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "batterer"
+	w_class = W_CLASS_TINY
+	flags = FPRINT
+	var/cooldown = 0
+	mech_flags = MECH_SCAN_FAIL
+
+/obj/item/device/loic_remote/process()
+	if(cooldown - world.time < 0)		//Set the icon back to the default if the cooldown expires
```

Comment:
do this as an update_icon() instead

`icon_state = "batterer[cooldown-world.time<0 ? "" : "burnt"]`

Reply:
`/obj/item/device/loic_remote/process()
	update_icon()

/obj/item/device/loic_remote/update_icon()
	icon_state = "batterer[cooldown-world.time<0 ? "" : "burnt"]"
`

like this?

Reply:
yes